### DISCUSSION TITLE
[BUG] Update IdeaCon file with API not change load effect

### DISCUSSION THREAD

**Discussion by paolobia** [idea-statica/ideastatica-public#378] [Help Q&A] [answered]
Hi,

open existing connection IdeaCon 
`client.OpenProject(conname);`
`client.UpdateConProjFromIOM(iomFilename, iomFilename + "r");`
but I have the old load!

 - OS: Windows 10-1903
 - IdeaStatiCa Version 23.1.0.4061

**Comment by nathanlukeIDEA** [comment]
Hi @paolobia is it possible to send the files so I can test?

**Comment by paolobia** [comment]
In attach the files
[idea.zip](https://github.com/idea-statica/ideastatica-public/files/13436973/idea.zip)

**Comment by paolobia** [comment]
<img width="325" alt="import load xls" src="https://github.com/idea-statica/ideastatica-public/assets/633707/00309b19-634f-4c6c-8a3b-fa9cf8d11035">

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @paolobia you need to call the `client.Save()` method after the update or else the update will not be saved back to the original file. 

So below should work.

```csharp
client.OpenProject(conname);
client.UpdateConProjFromIOM(iomFilename, iomFilename + "r");
client.Save();
client.CloseProject();
```

regarding excel import see https://www.ideastatica.com/support-center/how-to-import-load-effects-from-excel-sheet

**Reply by paolobia** [reply]
thank you !!!
It works!
